public/index.html

```diff
@@ -57,15 +57,43 @@
             
             <!-- Leave prod version commented for development -->
             <!-- Prod version -->
-            <li><a href="/faqs" class="anchor">FAQs</a></li>
+            <!-- <li><a href="/faqs" class="anchor">FAQs</a></li> -->
 
             <!-- Dev version -->
             <!-- <li><a href="faqs.html" class="anchor">FAQs</a></li> -->
-            
-            <li><a href='https://www.reddit.com/r/poap' class="anchor" target="_blank"><img src="assets/images/reddit2.svg" class="mlogo" alt="Reddit" /> Reddit</a></li>
-            <li><a href="https://github.com/poap-xyz" class="anchor"><img src="assets/images/logo-git2.svg" class="mlogo" alt="Github" />Github</a></li>
-            <li><a href="https://discord.gg/9s8U8Bn" class="anchor"><img src="assets/images/logo-discord2.svg" class="mlogo" alt="Discord" />Discord</a></li>
-            <li><a href="https://twitter.com/poapxyz" class="anchor"><img src="assets/images/logo-twitter2.svg" class="mlogo" alt="twitter" />@poapxyz</a></li>
+            <li><a href='https://app.poap.xyz/scan/' class="anchor">My collection</a></li>
+            <li><a href='https://app.poap.xyz/admin/' class="anchor">Mint a POAP</a></li>
```

Comment:
Links that point outside the website should open in a new tab

Reply:
Good catch! I'll make that change